examples/src/main/scala/org/apache/spark/examples/ml/QuantileDiscretizerExample.scala

```diff
@@ -45,7 +45,7 @@ object QuantileDiscretizerExample {
       .setNumBuckets(3)
 
     val result = discretizer.fit(df).transform(df)
-    result.show()
+    result.show(false)
```

Comment:
One more question - is it necessary to make this not truncate?

Reply:
We're following same style in other examples so it is good to do.

Reply:
Which other examples? most do not set this, and the Java equivalent doesn't either. If there's a good reason that the output needs to be untruncated, that's fine, just also change the Java example.

Reply:
@srowen correct either way it works for ex. examples/ml/LDAExamples.scala